sources/lib/api/gcp/bigquery/_query.py

```diff
@@ -26,7 +28,8 @@ class Query(object):
   """
 
   @staticmethod
-  def sampling_query(sql, context, fields=None, count=5, sampling=None):
+  def sampling_query(sql, context, fields=None, count=5, sampling=None, scripts=None,
+                     external_tables=None):
```

Comment:
So the docs seem to use Federated Data Sources (https://cloud.google.com/bigquery/federated-data-sources) here -- so maybe this can be datasources rather than external_tables in the parameter naming.